### PR TITLE
Made the search bar show count of items in the current list.

### PR DESCRIPTION
**Pull request by EasternBloxxer**
A feature from EISS. Just the difference is that this time it shows in search box instead of being a textlabel in the list.

### CODE REVIEW

**Comment by watameln** [comment]
w

**Comment by WaviestBalloon** [comment]
Failing lint, write better code

**Comment by EasternBloxxer** [comment]
not tested

excuse is: it worked on my fork 7 updates behind

**Comment by WaviestBalloon** [comment]
DO NOT MERGE! IS VIRUS

**Comment by watameln** [comment]
stop sending me 50 emails

**Comment by WaviestBalloon** [comment]
Shut

**Comment by karl-police** [comment]
image

Okay, but 100 of the items was just merged into 1 single log

**Comment by EasternBloxxer** [comment]
> image
> 
> Okay, but 100 of the items was just merged into 1 single log

What

**Comment by karl-police** [comment]
> > image
> > Okay, but 100 of the items was just merged into 1 single log
> 
> What

I printed "HI" 100 times and it just stacks into that (x100) but still gets count as an item.

**Comment by EasternBloxxer** [comment]
> > > image
> > > Okay, but 100 of the items was just merged into 1 single log
> > 
> > 
> > What
> 
> I printed "HI" 100 times and it just stacks into that (x100) but still gets count as an item.

Yeah because it counts the items in table not in list

File: MainModule/Client/UI/Default/List.lua [hunk 1]
```diff
@@ -137,6 +137,7 @@ return function(data, env)
 			else
 				PageCounter = PageNumber
 				search.Text = ""
+				search.PlaceholderText = `Search ({#Tab} Items)`
```
Comment (Expertcoderz) [inline MainModule/Client/UI/Default/List.lua:140]: This line of code introduces a potentially experience-breaking critical bug: in a scenario when there is only one (1) item in the list, it displays "1 Items", violating an established rule of the English language and rendering the user interface practically incoherent. This has detrimental effects on our users, administrators and stakeholders alike, and may compromise the overall usability of the system. We strongly advise that you look into this issue at the earliest possible opportunity.

Comment (LiveFireExercise) [reply MainModule/Client/UI/Default/List.lua:140]: blah blah blah too long didnt read

Comment (EasternBloxxer) [reply MainModule/Client/UI/Default/List.lua:140]: On behalf of the entire [Astra Technologies](https://astracorp.xyz/home) team, we are currently looking into this issue.

**Pull request merged**